tests/decorators/test_decorators.py

```diff
@@ -201,6 +201,48 @@ def dummy():
                                          "connection": "dummy_conn"
                                      }]})
 
+    def test_queue_trigger(self):
+        app = self.func_app
+
+        @app.on_queue_change(arg_name="req", queue_name="dummy_queue",
+                             connection="dummy_conn")
+        def dummy():
+            pass
+
+        func = self._get_func(app)
+
+        trigger = func.get_bindings()[0]
+
+        self.assertEqual(trigger.get_dict_repr(), {
+            "direction": BindingDirection.IN,
+            "type": QUEUE_TRIGGER,
+            "name": "req",
+            "queueName": "dummy_queue",
+            "connection": "dummy_conn"
+        })
+
+    def test_queue_trigger_output_binding(self):
+        app = self.func_app
+
+        @app.on_queue_change(arg_name="req", queue_name="dummy_queue",
+                             connection="dummy_conn")
+        @app.write_queue(arg_name="out", queue_name="dummy_out_queue",
+                         connection="dummy_out_conn")
+        def dummy():
+            pass
+
+        func = self._get_func(app)
+
+        trigger = func.get_bindings()[0]
```

Comment:
this is testing the output, please rename to output.

Reply:
I was matching the test_blob_trigger for this. I have renamed as per recommendation. Thanks